stats/stats-proto/src/tests.rs

```diff
@@ -0,0 +1,39 @@
+use prost::Message;
+
+use crate::blockscout::stats::v1::{self as proto};
+
+const PRECISE_POINT_1: &str = r#"
+{
+    "date": "2024-03-14",
+    "value": "188542399",
+    "isApproximate": false
+}
+"#;
+
+const PRECISE_POINT_2: &str = r#"
+{
+    "date": "2024-03-14",
+    "value": "188542399"
+}
+"#;
+
+#[test]
+fn is_approximate_serialization() {
```

Comment:
it looks like you are testing `serde_json` implementation. I think it's a little unnecessary, but you can leave it if you want

Reply:
more like integration between `actix_prost` and `serde_json` in this case)
yeah, the intention is to check if the field is serialized conditionally as expected; maybe it's possible to do it in some serializer-agnostic way (but rn it's simple and it works)